simon_says/manifests/turtlebot.json

```diff
@@ -0,0 +1,58 @@
+{
+  "apiVersion": "v1.0.0",
+  "name": "simon_says_turtlebot",
+  "packageVersion": "v1.0.0",
+  "description": "",
+  "plans": [
+    {
+      "name": "default",
+      "metadata": {},
+      "singleton": false,
+      "components": [
+        {
+          "name": "default",
+          "description": "",
+          "cloudInfra": {
+            "replicas": 1,
+            "endpoints": []
+          },
+          "ros": {
+            "topics": [
+              {
+                "name": "/turtlebot/status",
+                "qos": "low"
+              }
+            ],
+            "services": [],
+            "actions": [],
+            "isROS": true
+          },
+          "requiredRuntime": "device",
+          "executables": [
+            {
+              "cmd": [
+                "source /home/$USER/catkin_ws/devel/setup.bash && roslaunch turtlebot turtlebot.launch"
```

Comment:
you can remove this source line now since the new console will let us able to source the ROS setup scripts by mentioning the catkin workspace as device parameter.